mediaflow_proxy/speedtest/models.py

```
@@ -21,26 +20,20 @@ class UserInfo(BaseModel):
     country: Optional[str] = None
 
 
-class SpeedTestResult(BaseModel):
-    speed_mbps: float = Field(..., description="Speed in Mbps")
-    duration: float = Field(..., description="Test duration in seconds")
-    data_transferred: int = Field(..., description="Data transferred in bytes")
-    timestamp: datetime = Field(default_factory=datetime.utcnow)
+class MediaFlowServer(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)